tests/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixerTest.php

```diff
@@ -135,7 +135,12 @@ public function AB($self)
         return $this; // %s
      }
 }
-', $input, $input, $expected, $input);
+',
```

Comment:
This `sprintf` call and the one below were not multiline and did not need to be fixed. You should either revert these changes or add the missing newline after the opening brace.